src/config/constants/farms.ts

```diff
@@ -12,6 +12,2441 @@ const farms: FarmConfig[] = [
+    },
+    token: tokens.ramp,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 316,
+    lpSymbol: 'BFI-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x153Ad7d25B0b810497483d0cEE8AF42Fc533FeC8',
+    },
+    token: tokens.bfi,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 315,
+    lpSymbol: 'DEXE-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x3578B1f9BCE98D2F4D293b422d8850fdf48B1f21',
+    },
+    token: tokens.dexe,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 314,
+    lpSymbol: 'BEL-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x69DEE989c30b5fFe40867f5FC14F00E4bCE7B681',
+    },
+    token: tokens.bel,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 313,
+    lpSymbol: 'TPT-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x6D0c831254221ba121fB53fb44Df289A6558867d',
+    },
+    token: tokens.tpt,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 312,
+    lpSymbol: 'WATCH-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x13321AcfF4A27f3d2bcA64b8bEaC6e5FdAAAf12C',
+    },
+    token: tokens.watch,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 311,
+    lpSymbol: 'xMARK-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0xAa40f1AC20AAFcFEE8595Da606D78C503C7e70A3',
+    },
+    token: tokens.xmark,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 310,
+    lpSymbol: 'bMXX-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xc20A92a1424b29b78DFaF92FD35D4cf8A06419B4',
+    },
+    token: tokens.bmxx,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 309,
+    lpSymbol: 'IOTX-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0xc13aA76AAc067c86aE38028019F414D731b3D86A',
+    },
+    token: tokens.iotx,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 197,
+    lpSymbol: 'BOR-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xe094c686aD6cDda57b9564457F541FBF099B948A',
+    },
+    token: tokens.bor,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 307,
+    lpSymbol: 'bOPEN-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xc7A9c2af263ebB86139Cca9349e49b17129Ba033',
+    },
+    token: tokens.bopen,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 306,
+    lpSymbol: 'SUSHI-ETH LP',
+    lpAddresses: {
+      97: '',
+      56: '0x16aFc4F2Ad82986bbE2a4525601F8199AB9c832D',
+    },
+    token: tokens.sushi,
+    quoteToken: tokens.eth,
+  },
+  {
+    pid: 305,
+    lpSymbol: 'DODO-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xA9986Fcbdb23c2E8B11AB40102990a08f8E58f06',
+    },
+    token: tokens.dodo,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 303,
+    lpSymbol: 'BRY-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x4Fd6D315bEf387fAD2322fbc64368fC443F0886D',
+    },
+    token: tokens.bry,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 302,
+    lpSymbol: 'ZEE-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x21dD71aB78EDE3033c976948f769D506E4F489eE',
```

Comment:
I think this is the BRY-BNB address, it should be `0x8e799cB0737525CeB8A6C6Ad07f748535fF6377B`